config/initializers/time_formats.rb

```diff
@@ -0,0 +1,18 @@
+# Time Formats
+# ============
+#
+# Adds named date and time formats for time.to_s and date.to_s
+#
+# To display the current date and time in all the formats in your current environment:
+#
+#   Date::DATE_FORMATS.keys.push(:default).each{|k| puts "#{k}: #{Date.today.to_s(k)}"}
+#   Time::DATE_FORMATS.keys.push(:default).each{|k| puts "#{k}: #{Time.now.to_s(k)}"}
+#
+# For examples time = Time.local(2009,12,4,15,30,27)
+
+shared_formats = {
+  d_month_yyyy: "%-d %B %Y", # '4 December 2009'
+}
+
+Time::DATE_FORMATS.merge! shared_formats
+Date::DATE_FORMATS.merge! shared_formats
```

Comment:
Good idea - creating a common Date format. Do you know why we need to add it to Time and Date and not just Date?

Reply:
I wasn't sure which time type was being used in the code - so covered both.

Reply:
Also `Date` is a separate object and behaves a little differently to `Time`.